x-pack/plugins/cases/public/components/all_cases/use_cases_columns.tsx

```diff
@@ -58,6 +59,16 @@ const MediumShadeText = styled.p`
   color: ${({ theme }) => theme.eui.euiColorMediumShade};
 `;
 
+const LINE_CLAMP = 3;
+const LineClampedEuiBadgeGroup = euiStyled(EuiBadgeGroup)`
```

Comment:
As discussed, depending on the length of the tags, the ellipsis sometimes isn't displayed and the tooltip doesn't show more than 3 lines so additional tags are not shown in the tooltip.

For example here I have 9 tags added to the case:

<img width="422" alt="image" src="https://user-images.githubusercontent.com/7405507/200819015-302be9d6-f78c-46bc-b84b-b3f6d4024de5.png">

but only 6 are shown in the cases list tooltip:

<img width="532" alt="image" src="https://user-images.githubusercontent.com/7405507/200819170-af0982bf-04ea-40cf-a720-5c8dbcf38bdd.png">